app/controllers/application_controller.rb

```diff
@@ -20,4 +22,26 @@ def content_item_path
 
     "/" + URI.encode_www_form_component(path_and_optional_locale).gsub("%2F", "/")
   end
+
+  def save_this_page_enabled?
+    ENV["FEATURE_FLAG_SAVE_A_PAGE"] == "enabled"
+  end
+  helper_method :save_this_page_enabled?
+
+  def user_signed_in?
+    logged_in?
+  end
+  helper_method :user_signed_in?
```

Comment:
Why use this instead of just `logged_in?`

Reply:
Ah right, yeah I had this doing some other stuff when it was in the helper proper.
But your right I could just make a method from the concern a helper_method.

I'll do that now.